src/components/TableFilters/CreatorTableFilter.tsx

```diff
@@ -31,11 +26,12 @@ const CreatorTableFilter = (): JSX.Element => {
     const dispatchWithAutoAbort = useDispatchWithAutoAbort();
     const [page] = usePage();
     const [match] = useMatch();
-    const [ordering] = useOrdering('username');
```

Comment:
What was the use of this function? Can we expect any regression from this change?